File: sandbox/node/src/FaxLineAreaCodeGetDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();

const country = "US";
const state = undefined;
const province = undefined;
const city = undefined;

apiCaller.faxLineAreaCodeGet(
    country,
    state,
    province,
    city,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLine#faxLineAreaCodeGet:");
  console.log(error.body);
});

File: sandbox/ruby/src/FaxLineAreaCodeGetDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::FaxLineApi.new.fax_line_area_code_get(
        "US",
        {
            state: nil,
            province: nil,
            city: nil,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling FaxLine#fax_line_area_code_get: #{e}"
end
